tests/Value/SizeTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Sabberworm\CSS\Tests\Value;
+
+use PHPUnit\Framework\TestCase;
+use Sabberworm\CSS\Parsing\ParserState;
+use Sabberworm\CSS\Settings;
+use Sabberworm\CSS\Value\Size;
+
+/**
+ * @covers \Sabberworm\CSS\Value\Size
+ */
+final class SizeTest extends TestCase
+{
+    /**
+     * @return array<string, array{0: string}>
+     */
+    public static function provideUnit(): array
+    {
+        $units = [
+            'px', 'pt', 'pc',
+            'cm', 'mm', 'mozmm', 'in',
+            'vh', 'dvh', 'svh', 'lvh',
+            'vw', 'vmin', 'vmax', 'rem',
+            '%', 'em', 'ex', 'ch', 'fr',
+            'deg', 'grad', 'rad', 's', 'ms', 'turn', 'Hz', 'kHz',
+        ];
+
+        return \array_combine(
+            $units,
+            \array_map(
+                function (string $unit): array {
```

Comment:
We should also use the units as array keys in the outer array: If we don't do this, PHPUnit will should something like "… with data set 12" when running the tests (which is less helpful for debugging).

Reply:
That's what the `array_combine` does: uses the values from `$units` also as keys.  (Saves duplicating each unit as both a key and a value.)

Reply:
Ah, I see. Thanks for the explanation!